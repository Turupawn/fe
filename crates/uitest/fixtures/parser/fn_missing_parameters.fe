fn foo -> u8 {}

fn bar {
}

fn baz -> u8 {}

fn f<T> where T: U {}