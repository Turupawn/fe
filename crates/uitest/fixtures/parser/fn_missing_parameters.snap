---
source: crates/uitest/tests/parser.rs
expression: diags
input_file: crates/uitest/fixtures/parser/fn_missing_parameters.fe
---
error[1-0001]: expected function parameter list
  ┌─ fn_missing_parameters.fe:1:7
  │
1 │ fn foo -> u8 {}
  │       ^ expected `(`

error[1-0001]: expected function parameter list
  ┌─ fn_missing_parameters.fe:3:7
  │
3 │ fn bar {
  │       ^ expected `(`

error[1-0001]: expected function parameter list
  ┌─ fn_missing_parameters.fe:6:7
  │
6 │ fn baz -> u8 {}
  │       ^ expected `(`

error[1-0001]: expected function parameter list
  ┌─ fn_missing_parameters.fe:8:8
  │
8 │ fn f<T> where T: U {}
  │        ^ expected `(`


